css-two-column-layout/styles.css

```diff
@@ -0,0 +1,54 @@
+.row {
+  display: flex;
+  justify-content: space-between;
+  max-width: 1200px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 50px;
+}
+
```

Comment:
You should be using the font we supplied for this challenge. So I would create a new selector for the `body` and apply `font-family: Karla, sans-serif`